couchdb-exporter.go

```diff
@@ -124,6 +125,13 @@ func init() {
 			Hidden:      false,
 			Destination: &exporterConfig.databaseViews,
 		}),
+		altsrc.NewUintFlag(cli.UintFlag{
+			Name:        "concurrency",
```

Comment:
We need to discuss this name. I'd prefer a more specific name, something along the idea of `concurrent-database-requests`.

Reply:
... or even `database-concurrent-requests`, to keep the name similar to the other parameters.

Reply:
Sure, I'll make a commit when I get time.